survey.go

```diff
@@ -175,6 +179,22 @@ func WithKeepFilter(KeepFilter bool) AskOpt {
 	}
 }
 
+// WithRemoveSelectAll remove the select all option in Multiselect
+func WithRemoveSelectAll(RemoveSelectAll bool) AskOpt {
```

Comment:
I think given that the name of the option is "Remove..." it's a bit confusing to accept a parameter. Does saying Remove(false) turn it on? I think it's probably less confusing if we don't accept any arguments.

Reply:
Ahh, yes I totally agree with this (I guess that shows through the docs being mixmatched) I think I was following the method above `WithKeepFilter(KeepFilter bool)`.

Updated these both to no longer have a bool as an arg here